ArtOfIllusion/src/artofillusion/PluginRegistry.java

```diff
@@ -41,19 +44,19 @@ Changes copyright (C) 2018-2022 by Maksim Khramov
    * contained in them.
    */
 
-  public static void scanPlugins()
```

Comment:
You're changing a public method signature here without warning. What is the expected deprecation/upgrade pathway like?

Reply:
Only use for this method is to start plugins scan at application start. I did not find any other usages for this so I think it is safe enough to change. Moreover if someone want to call this method it is safe to ignore return value